thz/thz.yaml

```diff
@@ -0,0 +1,856 @@
+    # [[p24Hyst4]]
+    # type = num
+    # thz = p24Hyst4
+    # [[p25Hyst5]]
+    # type = num
+    # thz = p25Hyst5
+    # [[p29HystAsymmetry]]
+    # type = num
+    # thz = p29HystAsymmetry
+    # [[p30integralComponent]]
+    # type = num
+    # thz = p30integralComponent
+    # [[p32hystDHW]]
+    # type = num
+    # thz = p32hystDHW
+    # [[p33BoosterTimeoutDHW]]
+    # type = num
+    # thz = p33BoosterTimeoutDHW
+    # [[p34BoosterDHWTempAct]]
+    # type = num
+    # thz = p34BoosterDHWTempAct
+    # [[p35PasteurisationInterval]]
+    # type = num
+    # thz = p35PasteurisationInterval
+    # [[p35PasteurisationTemp]]
+    # type = num
+    # thz = p35PasteurisationTemp
+    p37FanStage1AirIn:
+        type: num
+        thz: p37FanStage1AirIn
+
+    # [[p38FanStage2AirIn]]
+    # type = num
+    # thz = p38FanStage2AirIn
+    # [[p39FanStage3AirIn]]
+    # type = num
+    # thz = p39FanStage3AirIn
+    p40FanStage1AirOut:
+        type: num
+        thz: p40FanStage1AirOut
+    # [[p41FanStage2AirOut]]
+    # type = num
+    # thz = p41FanStage2AirOut
+    # [[p42FanStage3AirOut]]
+    # type = num
+    # thz = p42FanStage3AirOut
+    # [[p43UnschedVent3]]
+    # type = num
+    # thz = p43UnschedVent3
+    # [[p44UnschedVent2]]
+    # type = num
+    # thz = p44UnschedVent2
+    # [[p45UnschedVent1]]
+    # type = num
+    # thz = p45UnschedVent1
+    # [[p46UnschedVent0]]
+    # type = num
+    # thz = p46UnschedVent0
+    # [[p49SummerModeTemp]]
+    # type = num
+    # thz = p49SummerModeTemp
+    # [[p50SummerModeHysteresis]]
+    # type = num
+    # thz = p50SummerModeHysteresis
+    # [[p54MinPumpCycles]]
+    # type = num
+    # thz = p54MinPumpCycles
+    # [[p55MaxPumpCycles]]
+    # type = num
+    # thz = p55MaxPumpCycles
+    # [[p56OutTempMaxPumpCycles]]
+    # type = num
+    # thz = p56OutTempMaxPumpCycles
+    # [[p57OutTempMinPumpCycles]]
+    # type = num
+    # thz = p57OutTempMinPumpCycles
+    # [[p75passiveCooling]]
+    # type = num
+    # thz = p75passiveCooling
+    # [[p76RoomThermCorrection]]
+    # type = num
+    # thz = p76RoomThermCorrection
+    # [[p77OutThermFilterTime]]
+    # type = num
+    # thz = p77OutThermFilterTime
+    # [[p78DualModePoint]]
+    # type = num
+    # thz = p78DualModePoint
+    # [[p79BoosterTimeoutHC]]
+    # type = num
+    # thz = p79BoosterTimeoutHC
+    # [[p83DHWsetSolarTemp]]
+    # type = num
+    # thz = p83DHWsetSolarTemp
+    # [[p99DHWmaxFlowTemp]]
+    # type = num
+    # thz = p99DHWmaxFlowTemp
+    # [[p89DHWeco]]
+    # type = num
+    # thz = p89DHWeco
+    # [[p99startUnschedVent]]
+    # type = num
+    # thz = p99startUnschedVent
+    # [[pClockDay]]
+    # type = num
+    # thz = pClockDay
+    # [[pClockDay]]
+    # type = num
+    # thz = pClockDay
+    # [[pClockYear]]
+    # type = num
+    # thz = pClockYear
+    # [[pClockHour]]
+    # type = num
+    # thz = pClockHour
+    # [[pClockMinute]]
+    # type = num
+    # thz = pClockMinute
```

Comment:
What is this YAML file for? Only for documentation purpose? Maybe this all can be moved into the `plugin.yml` to section `item_attributes` to document them.